invokeai/backend/stable_diffusion/diffusion/custom_atttention.py

```diff
@@ -9,6 +10,11 @@
 from invokeai.backend.stable_diffusion.diffusion.regional_prompt_data import RegionalPromptData
 
 
+class IPAdapterAttentionWeights(TypedDict):
+    ip_adapter_weights: List[IPAttentionProcessorWeights]
+    skip: bool
```

Comment:
Nitpick, using a dataclass feels cleaner to me bc you can just use `ip_adapter_attention_weights.ip_adapter_weights` instead of `ip_adapter_attention_weights["ip_adapter_weights"]` but it's not a big deal.

Reply:
I'll update that.